### DISCUSSION TITLE
PatchTool 6.3 1/11 修正

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#271] [Announcements]
[v2.72.0](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.72.0)的修正版。

檔案：

- FFXIVChnTextPatch-20230111-v2.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：

- rawexd-zhcn-20230111-v2.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 https://github.com/GpointChen/FFXIVChnTextPatch-GP/issues/90 放到與exe同位置使用。
- [backup-20230111.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.72.0/backup-20230111.rar)：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

目前測試，在之前的更新後，第三方4k字體包可能會導致遊戲崩潰。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.72.1'>PatchTool 6.3 1/11 修正</a>.</em>

**Comment by cream313** [comment]
興上一版本同樣 4-8層零式仍然未修正 謝謝 感謝更新

**Reply by GpointChen** [reply]
目前看來是從中國服提取出來的文本本身就有出現問題，會出現如下圖的一串英數字。

![](https://user-images.githubusercontent.com/34565875/211694347-ac77d45a-607a-45bc-8e7f-3fb4d00d6569.png)

不確定這個問題的原因，待後續確認。

**Comment by rorocochi** [comment]
您好，我這邊初次安裝漢化器，執行exe檔後只會跳出 "您是否要允許這個來自未知發行者的APP變更您的裝置?" 按是之後就無反應了，沒有看到漢化設置程式執行的畫面，希望幫解惑 謝謝！

**Reply by GpointChen** [reply]
聽起來很神奇

你是Windows嗎？電腦裡有安裝任何版本的Java嗎？

**Reply by rorocochi** [reply]
謝謝回覆 我的電腦是Windows10版本，剛剛查看了一下應該是沒有安裝任何Java

**Reply by GpointChen** [reply]
那你可能需要去下載16.0.1以上的Java，可以去[這裡](https://jdk.java.net/archive/)下載。

**Reply by rorocochi** [reply]
已經下載了，不好意思請問下載過後要把JAVA資料夾存放在哪裡呢?

**Reply by GpointChen** [reply]
存放在哪裡都可以，但你需要去系統環境變數添加路徑。詳細可以參考這篇：https://jakingfox.blogspot.com/2019/01/windowsopenjdk.html?m=1

**Reply by rorocochi** [reply]
您好我有照著參考裡的做法做好了，也有用命令提示字元確定有安裝了JAVA 19.0.2的版本
不過回頭跑漢化程式仍然是跟之前一樣的狀況😅 不好意思再次麻煩大大了 謝謝您的耐心

**Reply by GpointChen** [reply]
你參考看看 #90 

下載我貼的那個zip檔案，然後解壓到跟exe同位置然後打指令看看會有什麼結果

**Reply by rorocochi** [reply]
這次有成功了! 謝謝大大耐心解答 那麼以後如果要更新漢化是否都是一樣透過這樣的方式開啟呢?
